src/components/selects-request/archive-list/table/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { Table } from 'antd';
+
+import { HEADER_HEIGHT } from '@src/component/organisms/g-header';
+
+function ArchiveTable({ dataSource, colums, dataMapper = (data) => data }) {
+  if (dataSource == undefined) {
+    console.log(dataSource);
+    return null;
+  }
```

Comment:
```suggestion
  if (!dataSource) {
    console.log(dataSource);
    return null;
  }
```

null일때도 대응되도록 위와 같이 변경해도 좋을 것 같아요

Reply:
3cea873